skimage/filters/tests/test_gaussian.py

```diff
@@ -82,3 +89,14 @@ def test_guess_spatial_dimensions():
     testing.assert_equal(_guess_spatial_dimensions(im4), 3)
     with testing.raises(ValueError):
         _guess_spatial_dimensions(im5)
+
+
+@pytest.mark.parametrize(
+    "dtype", [np.float32, np.float64]
+)
+def test_preserve_output(image, dtype):
```

Comment:
I would get rid of the fixture. It isn't providing anything more than a convenience, (no speed increase), and hides the fact that it is a number between 0 and 10.

Reply:
Very good point, hard to disagree.